docker/tasks/main.yml

```diff
@@ -27,8 +26,12 @@
 - name: Docker repository
   become: yes
   ansible.builtin.apt_repository:
-    repo: deb [arch=amd64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu {{ ansible_distribution_release }} stable
-  when: "ansible_architecture == 'arm64'"
+    repo: deb [arch=arm64 signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu {{ ansible_distribution_release }} stable
+  when: "ansible_architecture == 'aarch64'"
+
+- name: Update apt packages
+  become: yes
+  shell: apt-get update
```

Comment:
Should use the apt module for this, right?

```
- name: Run the equivalent of "apt-get update"
  apt:
    update_cache: yes
```

Reply:
In fact you could probably also just delete this task since you have `update_cache` in the step immediately following this.

Reply:
Changed to the module.